CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+All notable changes to this module will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this module adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+
+## [1.1.0] - 2024-03-04
+
+One-shot deployments only, no new features, improved templates.
+
+### Added
+
+- Tags module for subscription in main.tf template.
+- Subscription local parameters
```

Comment:
```suggestion
- Tags module for subscription in main.tf template.
```
You don't need to write down internal changes

Reply:
Resolves as suggest.